components/announcement_bar/contact_sales/contact_sales.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import './contact_sales.scss';
+
+export interface Props {
+    buttonTextElement: JSX.Element;
+    eventID?: string;
+}
+
+const ContactSales: React.FC<Props> = (props: Props) => {
+    const handleContactSalesLinkClick = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+        e.preventDefault();
+        trackEvent('admin', props.eventID || 'in_trial_contact_sales');
+        window.open('https://mattermost.com/contact-us/', '_blank');
```

Comment:
Since the link isn't leading to anything specific to sales but rather a general contact us form,
should we consider renaming this component and repurposing it as a `ContactUs` component where you can change the text it displays but ultimately leads to the same form?

Reply:
I noticed on this website the inquiry types ... maybe if it accepts query parameters to fill those fields it, we can have props that you can pass to this component that fill those query params (I've asked in developers channel)

Reply:
1. Done the part of having a default text in the component itself and allowing the consumer to override it.
2. For the pre-filled form, we need some way to collect those inputs. So collecting them in Mattermost then passing them over to the contact us page would be the same for the user as they are effectively entering the information once. So I think we can pass this.

Reply:
Let me know if the second point makes sense if you have some idea.